presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlCaseInsensitiveMapping.java

```diff
@@ -100,7 +74,7 @@ public void testNonLowerCaseTableName()
         try (AutoCloseable ignore1 = withSchema("SomeSchema");
                 AutoCloseable ignore2 = withTable(
                         "SomeSchema.NonLowerCaseTable", "(lower_case_name varchar(10), Mixed_Case_Name varchar(10), UPPER_CASE_NAME varchar(10))")) {
-            execute("INSERT INTO SomeSchema.NonLowerCaseTable VALUES ('a', 'b', 'c')");
+            mySqlServer.execute("INSERT INTO SomeSchema.NonLowerCaseTable VALUES ('a', 'b', 'c')");
```

Comment:
Same comment as `TestMySqlIntegrationSmokeTest`. Let's keep the `execute` method for now.